pages/glossary/programmatically-determined-link-context.md

```diff
@@ -12,16 +12,20 @@ The _programmatically determined context_ of a link (or _programmatically determ
 
 - being an [ancestor][] of the link in the [flat tree][] with a [semantic role][] of `listitem`; or
 - being the closest [ancestor][] of the link in the [flat tree][] that is a `p` element; or
```

Comment:
That one might be removable in favour of the next one.
Even if `p` are styled with `display: line`, I'd say they might not be enough to  be a context by themselves and need to be wrapped in the closest block 🤔

Reply:
Agreed.